osu.Game.Rulesets.Catch/Skinning/Legacy/LegacyDropletPiece.cs

```diff
@@ -2,12 +2,15 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using osu.Framework.Graphics.Textures;
+using osu.Game.Skinning;
 using osuTK;
 
 namespace osu.Game.Rulesets.Catch.Skinning.Legacy
 {
     public partial class LegacyDropletPiece : LegacyCatchHitObjectPiece
     {
+        private static readonly Vector2 droplet_max_size = new Vector2(100);
```

Comment:
Where does 100 come from? The template skin is 82 x 103 (`@1x`).

Reply:
Not to say it's correct, but I think I picked 100 as it's closest to 103...I'll change to 82x103